testgrid/config.yaml

```diff
@@ -3847,78 +3847,113 @@ dashboards:
+  - name: kubeadm-gce-1.13-on-1.14
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-13-on-1-14
+  - name: kubeadm-gce-stable-on-master
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-stable-on-master
+    alert_options:
+      alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
+    alert_stale_results_hours: 24
+    num_failures_to_alert: 4 # Runs every 6h. Alert when it's been failing for 24 hours
+# kubeadm upgrade tests
+  - name: kubeadm-gce-upgrade-1.11-1.12
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-11-1-12
+  - name: kubeadm-gce-upgrade-1.12-1.13
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-12-1-13
+  - name: kubeadm-gce-1.13-1.14
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-13-1-14
+  - name: kubeadm-gce-upgrade-stable-master
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-upgrade-stable-master
+# packages tests
+  - name: periodic-packages-pushed
+    test_group_name: periodic-kubernetes-e2e-packages-pushed
+  - name: periodic-packages-install-deb
+    test_group_name: periodic-kubernetes-e2e-packages-install-deb
+# manifest list tests
+  - name: periodic-manifest-lists
+    test_group_name: periodic-kubernetes-e2e-manifest-lists
+
+- name: sig-cluster-lifecycle-kubeadm
+  dashboard_tab:
 # kubeadm-kind tests
   - name: kubeadm-kind-1.12
     test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-12
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 24
-    num_failures_to_alert: 6 # Runs every 4h. Alert when it's been failing for 24 hours
+    num_failures_to_alert: 2 # Runs every 12h. Alert when it's been failing for 24 hours
   - name: kubeadm-kind-1.13
     test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-13
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 24
-    num_failures_to_alert: 6 # Runs every 4h. Alert when it's been failing for 24 hours
+    num_failures_to_alert: 2 # Runs every 12h. Alert when it's been failing for 24 hours
   - name: kubeadm-kind-1.14
     test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-14
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 24
-    num_failures_to_alert: 6 # Runs every 4h. Alert when it's been failing for 24 hours
+    num_failures_to_alert: 2 # Runs every 12h. Alert when it's been failing for 24 hours
   - name: kubeadm-kind-master
     test_group_name: ci-kubernetes-e2e-kubeadm-kind-master
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
-    alert_stale_results_hours: 24
-    num_failures_to_alert: 12 # Runs every 1h. Alert when it's been failing for 12 hours
+    alert_stale_results_hours: 12
+    num_failures_to_alert: 3 # Runs every 4h. Alert when it's been failing for 12 hours
```

Comment:
if you like my suggestion to move this to 2h, this line should be changed to:
`num_failures_to_alert: 6 # Runs every 2h. Alert when it's been failing for 12 hours`

Reply:
switched to 2h alert when failing 8h